backend/src/apiserver/common/config.go

```diff
@@ -32,6 +32,10 @@ const (
 	KubeflowUserIDPrefix                    string = "KUBEFLOW_USERID_PREFIX"
 	UpdatePipelineVersionByDefault          string = "AUTO_UPDATE_PIPELINE_DEFAULT_VERSION"
 	TokenReviewAudience                     string = "TOKEN_REVIEW_AUDIENCE"
+	MetadataTLSEnabled                      string = "METADATA_TLS_ENABLED"
+	CaBundleMountPath                       string = "CABUNDLE_MOUNTPATH"
+	CaBundleCertName                        string = "CABUNDLE_CERT_NAME"
```

Comment:
Why are these separate configurations rather than a single configuration that is a path to the bundle?

Reply:
@HumairAK do you have context on this?